src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -2388,7 +2388,7 @@ def addTornadoOptionsToCS(self, gConfig_o):
             return S_ERROR(f"{instanceOption} not defined in {self.cfgFile}")
         tornadoSection = cfgPath("Systems", "Tornado", compInstance)
 
-        cfg = self.__getCfg(tornadoSection, "Port", 8443)
+        cfg = self.__getCfg(tornadoSection, "Port", gConfig_o.getValue(cfgPath(tornadoSection, "Port"), "8443"))
```

Comment:
The rest of the function gives one the impression that gConfig_o could be `None` or something, so this might fail hard.